lib/new_relic/agent/noticeable_error.rb

```diff
@@ -7,7 +7,7 @@ module Agent
     # This class is used for instrumentations that have exceptions or error classes
     # not derived from Ruby's usual Exception or StandardError or in situations
     # where we do not have such Exception object to work with.
-    class NoticibleError
+    class NoticeableError
```

Comment:
File rename was called out above

Reply:
I love this feature!

Reply:
File renames are done manually. I have a script that handles it w/ the same inputs as the corrections, but I don't run it automatically. I'm not sure why I haven't integrated it into the baseline (I think it's just laziness, although it would add a slight performance cost to each replacement as it would have to think about applying it to the file set.).